packages/opentelemetry/lib/index.js

```diff
@@ -13,20 +18,44 @@ const logger = require('@dotcom-reliability-kit/logger');
  * @typedef {object} Options
  * @property {string} [authorizationHeader]
  *      [DEPRECATED] The HTTP `Authorization` header to send with OpenTelemetry requests. Use `tracing.authorizationHeader` instead.
+ * @property {MetricsOptions} [metrics]
+ *      Configuration options for OpenTelemetry metrics.
  * @property {TracingOptions} [tracing]
  *      Configuration options for OpenTelemetry tracing.
  */
 
+/**
+ * @typedef {object} Instances
+ * @property {opentelemetry.NodeSDK} sdk
+ *      A singleton instance of the OpenTelemetry Node SDK.
+ * @property {HostMetrics} [hostMetrics]
+ *      A singleton instances of the OpenTelemetry host metrics collector.
+ */
+
+/**
+ * Stores the singleton instances that were created during OpenTelemetry setup.
+ *
+ * @type {Instances | undefined}
+ */
+let instances;
+
 /**
  * Set up OpenTelemetry tracing.
  *
  * @param {Options} [options]
  *      OpenTelemetry configuration options.
+ * @returns {Instances}
+ *      Returns any created SDK instances.
  */
 function setupOpenTelemetry({
 	authorizationHeader,
+	metrics: metricsOptions,
 	tracing: tracingOptions
 } = {}) {
+	if (instances) {
+		return instances;
+	}
```

Comment:
**note:** I know that OpenTelemetry itself has some checks for this kind of thing, but I wanted to be sure that accidentally setting up otel twice doesn't cause weird issues. This also allows us to check whether OpenTelemetry has been set up via Reliability Kit before allowing other operations.